woocommerce/woocommerce-android
# Feature/9820 persist selected ai tone

Closes: #9820 


### Description

Saves selected AI tone into preference.

### Testing instructions

1. Trigger create product with AI flow
2. Navigate to "About your product" step
3. Select an AI tone different from `Casual`
4. Kill the app
5. Repeat steps 1 to 3 and check that when we open the bottom sheet the previously selected tone is selected. 

PS: I believe for now saving selected AI tone per user session/login should be enough. I think that is unlikely the case were users want to have different AI tones across different stores linked to the same account. 

### Images/gif


(Ignore the color issues from the demo. Those were already fixed in the linked PR)

https://github.com/woocommerce/woocommerce-android/assets/2663464/bf293cfa-aee7-424e-b6da-2d756ec448da
